plugins/logrotator/README.md

```
@@ -0,0 +1,121 @@
+      cron: '10 * * * *', // custom cron, or use interval
+    },
+    async task() {
+      await rotator.rotate();
+    },
+  };
+};
+
+function getRotator(app) {
+  class CustomRotator extends app.LogRotator {
+    // return map that contains a pair of srcPath and targetPath
+    // LogRotator will rename ksrcPath to targetPath
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)